src/annotator/config/settings.js

```diff
@@ -45,13 +45,13 @@ export default function settingsFrom(window_) {
    * Return the href URL of the first annotator client link in the given document.
    *
    * Return the value of the href attribute of the first
-   * `<link type="application/annotator+html" rel="hypothesis-client">` element in the given document.
+   * `<link type="application/annotator+html" rel="hypothesis-client">`
```

Comment:
```suggestion
   * `<link type="application/annotator+javascript" rel="hypothesis-client">`
```

This existing comment is out of sync with the code below.

Reply:
Fixed.